import_jobs.py

```
@@ -118,7 +118,7 @@ def get_jobsfs_zipfile(guid):
     authheader = "Basic %s" % base64.encodestring('%s:%s' % ('microsites',
                                                              'di?dsDe4'))
     req.add_header("Authorization", authheader)
-    resp = urllib2.urlopen(req)
+    resp = urllib2.urlopen(req, timeout=30)
```

No.  We're currently using the default timeout, but we're getting strange hanging behavior.  If a timeout occurs, it should raise an exception, which should result in the task being cancelled and retried in 3 minutes.  Hopefully, by then whatever is causing connection issues should be resolved.